docs/test-results/badges-browser-matrix.md

```diff
@@ -8,7 +8,9 @@ import useBaseUrl from '@docusaurus/useBaseUrl';
 
 Status badges and the Sauce Labs browser matrix widget help you keep track of the status of your latest test runs. All you have to do is add either markdown or HTML code to your GitHub README or project site that references your Sauce Labs username and access key, and annotate your tests with the REST API.
 
-## Status Badges and the Browser Matrix
+## Open Sauce Accounts
+Open Sauce accounts are public, meaning tests and build status can be viewed without authentication. To use badges with private accounts, see [Private Accounts](https://docs.saucelabs.com/test-results/badges-browser-matrix/#private-accounts).
```

Comment:
```suggestion
Open Sauce accounts are public, meaning tests and build status can be viewed without authentication. To use badges with private accounts, see [Private Accounts](#private-accounts).
```

Reply:
@lauramthomas I wasn't able to find this section in the doc: https://docs.saucelabs.com/test-results/badges-browser-matrix/**#private-accounts**

Reply:
Hey @lauramthomas -- I wasn't commenting on the section...I was just making it an internal link instead of an absolute link. In general, we should not be using `https://docs.saucelabs.com` prefix in our doc links. It causes an unnecessary reload of our site, which is a performance hit. Internal links should always be relative, and in the case of the one above, the link is actually to the same page, so you could simply use`#private-accounts`, except that you are right, and that heading does not seem to exist on this page, so I guess that is a more relevant fix than just the link relativity. :)